src/Headers.php

```diff
@@ -28,10 +27,29 @@ class Headers implements Countable, Iterator
     /** @var string Start of Line when folding */
     const FOLDING = "\r\n ";
 
-    /**
-     * @var \Zend\Loader\PluginClassLoader
-     */
-    protected $pluginClassLoader = null;
+    const HEADERS_CLASS_MAP = [
```

Comment:
This is not configurable.
Please move this into a variable in separate class with methods `get($name)`, `has($name)`, `add($name, $class)`, `remove($name)`

Reply:
I didn't realize it had to be configurable. I'll fix it right away.

Reply:
Do we want the new class to be exposed via getter as the original loader used to be?